05/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Website</title>
+</head>
+
+<body>
+    <nav class="nav">
+        <div class="cointeiner">
+            <ul class="nav__item">
+                <div class="nav__logo"></div>
+                <div class="nav__btn">
+                  menu
+                </div>
+                <ul class="nav__menu">
```

Comment:
Ten element nie powinien być widoczny (patrz obraz). Prawdopodobnie nie zresetowałeś ustawień dla `li` i to są wypunktowania ;)

![image](https://user-images.githubusercontent.com/55095257/233266100-b47d9102-4e1f-4a9b-8019-1993a09572ad.png)

Reply:
uznałem to za szczegół. już poprawiam